nautobot/docs/additional-features/graphql.md

```diff
@@ -127,4 +127,12 @@ Result
 }
 ```
 
+## Saved Queries
```

Comment:
Would you be willing to start a `nautobot/docs/release-notes/version-1.1.md` and add something similar to this content to that document?

Reply:
Sure can do this in another PR

Reply:
Can this be done as part of this PR?